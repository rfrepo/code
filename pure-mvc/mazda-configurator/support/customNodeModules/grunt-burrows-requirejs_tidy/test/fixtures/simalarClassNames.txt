define([
    'support/class',
    'support/class1'
], function () {
    'use strict';
    puremvc.define({
        bmc.support.class1.CONFIGURABLE_ITEM_UI;
    });
});