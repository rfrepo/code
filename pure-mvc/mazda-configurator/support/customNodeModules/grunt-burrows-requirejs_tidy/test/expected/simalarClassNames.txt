define([
    'support/class',
    'support/class1'
], function () {
    'use strict';
    var class1 = arguments[1];

    puremvc.define({
        class1.CONFIGURABLE_ITEM_UI;
    });
});